pkgs/applications/networking/cluster/helm2/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildGoModule, fetchFromGitHub, installShellFiles, Security }:
+
+buildGoModule rec {
+  pname = "helm";
+  version = "2.16.3";
```

Comment:
Could you please update it to 2.16.4 as it was released few days ago - https://github.com/helm/helm/releases/tag/v2.16.4

Reply:
Good catch, I never expected there will be a backport to helm 2 again

Reply:
It's actually one more released already :) https://github.com/helm/helm/releases/tag/v2.16.5